<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualVisitsDialog</class>
 <widget class="QDialog" name="ManualVisitsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,2">
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="workerlabel">
       <property name="text">
        <string>Worker</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="workersComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="doorlabel">
       <property name="text">
        <string>Door</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="doorsComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="datelabel">
       <property name="text">
        <string>Date</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="timelabel">
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTimeEdit" name="timeEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="useCurrentCheckBox">
       <property name="text">
        <string>Use Current</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="getThroughButton">
       <property name="text">
        <string>Get Through!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2" rowspan="5">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>446</width>
        <height>387</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
